readme.md

```diff
@@ -10,248 +8,408 @@ See also: https://github.com/ga-wdi-lessons/ruby-oop
-method. Talk with your partner for 7 minutes and discuss what effect each line
-will have, and which one is the best.
+But first, let's review and build upon some of the concepts that were introduced this morning...
 
-Feel free to run this code and play with it.
+## We Do: Let's Build A Class
 
-```ruby
-# person/self.rb
-class Person
-  attr_accessor :name
-  attr_reader :hunger_level
+Let's create a `Person` class...
 
-  def initialize(initial_name, initial_hunger_level)
-    @name = initial_name
-    @hunger_level = initial_hunger_level
+--------
+
+<details>
+  <summary><strong>What's the difference between a class and an instance?</strong></summary>
+  <br/>
+
+  A class is a blueprint. An instance is an object generated from that blueprint.
+  * A class' name is capitalized (e.g., `Person`) and is a template for objects of that class.
+  * An instance is one specific object created using a class (e.g. `bob = Person.new("Bob", 10)`).
+</details>
+<br/>
+<details>
+  <summary><strong>How do we begin defining a <code>Person</code> class?</strong></summary>
+
+  ```rb
+  class Person
   end
+  ```
+
+</details>
+
+--------
 
-  def hunger_level=(new_hunger_level)
-    if new_hunger_level < 0
-      @hunger_level = 0
-    else
-      @hunger_level = new_hunger_level
+<details>
+  <summary><strong>How can we set up our Person class so that it has <code>name</code> and <code>age</code> attributes. We should be able to read and modify <code>name</code>, but only read <code>age</code>.</strong></summary>
+
+  ```rb
+  class Person
+    # This allows us to both read and write to the name and hunger_level attributes
+    attr_accessor :name
+    attr_reader :age
+
+    # This is run every time we call Person.new
+    # Think of it as a "constructor method"
+    def initialize(initial_name, initial_age)
+      @name = initial_name
+      @age = initial_age
     end
   end
+  ```
+
+  > In place of the `attr` methods, we could also define getter and setter methods.
+
+</details>
+<br/>
+<details>
+  <summary><strong>Why did we place an <code>@</code> in front of our variables? How else could we have written our variables?</strong></summary>
+  <br/>
+
+  `@` indicates that we are defining an instance variable, which is a value that is accessible anywhere inside a given instance of a class.
+
+  The different types of variables we can use are...
+
+
+  #### Local
+
+  - Written using plain vanilla names (e.g., `current_age`, `favorite_color`).
+  - Limited in scope. Only accessible in the current method.
+  - Used for temporary values in a class.
+
+  #### Instance
+
+  - Start with an `@` (e.g., `@name`).
+  - Used to store attributes for a given instance.
+  - Can be used in any instance method in the class.
+  - Very common in ruby.
+
+  #### Class
+
+  - Start with 2 `@`s (e.g., `@@person_count`).
+  - One copy shared between all instances.
+  - Not very common. Over-use is considered bad practice.
```

Comment:
re: your question about why use class vars - nearly every example i've seen involves counting the instances created from the class e.g. `@@num_people`